lib/spack/spack/cmd/install.py

```diff
@@ -195,6 +198,11 @@ def add_cdash_args(subparser, add_help):
         default=None,
         help=cdash_help['site']
     )
+    subparser.add_argument(
+        '--cdash-authtoken',
```

Comment:
If we're taking this from a file, I think this should be called `--cdash-authtoken-file` or something like that.